src/Library/Platform/Sdl/SdlEventLoop.cpp

```diff
@@ -67,6 +68,7 @@ void SdlEventLoop::dispatchEvent(PlatformEventHandler *eventHandler, const SDL_E
 #ifdef MM_PLATFORM_SEND_NATIVE_EVENTS
     dispatchNativeEvent(eventHandler, event);
 #endif
+    ImGui_ImplSDL2_ProcessEvent(event);
```

Comment:
ImGui comes with several backends for managing events. I could have created a custom event handler to manage imgui but then I'd need to translate the Platform events back to ImGui and I think it's not worth it.

Reply:
We have `MM_PLATFORM_SEND_NATIVE_EVENTS` exactly for this purpose. Hook into it.

You can also drop the `#ifdef`s as now we'll be using native events unconditionally.

If you want to filter out the native events (so that they are not propagated to the rest of our machinery) then this is also doable, let's discuss in Discord.